scripts/exgfs_atmos_fbwind.sh

```diff
@@ -34,20 +34,17 @@ echo " "
 set_trace
 
 export pgm=bulls_fbwndgfs
-. prep_step
-
-for fhr in 006 012 024
-do
-
-  cp $COMIN/gfs.${cycle}.pgrb2.0p25.f${fhr}   tmp_pgrb2_0p25${fhr} 
-  cp $COMIN/gfs.${cycle}.pgrb2b.0p25.f${fhr}  tmp_pgrb2b_0p25${fhr} 
-  cat tmp_pgrb2_0p25${fhr} tmp_pgrb2b_0p25${fhr} > tmp0p25filef${fhr} 
-  $WGRIB2 tmp0p25filef${fhr} | grep  -F -f ${PARMgfs}/product/gfs_fbwnd_parmlist_g2 | $WGRIB2 -i -grib tmpfilef${fhr} tmp0p25filef${fhr}
-  $CNVGRIB -g21 tmpfilef${fhr} tmpfilef${fhr}.grib1
-  $GRBINDEX tmpfilef${fhr}.grib1 tmpfilef${fhr}.grib1i
-  mv tmpfilef${fhr}.grib1   gfs.t${cyc}z.grbf${fhr}_grb1
-  mv tmpfilef${fhr}.grib1i  gfs.t${cyc}z.grbif${fhr}_grb1
-
+source prep_step
+
+for fhr3 in 006 012 024; do
+  cp "${COMIN_ATMOS_GRIB_0p25}/gfs.${cycle}.pgrb2.0p25.f${fhr3}"   "tmp_pgrb2_0p25${fhr3}" 
```

Comment:
Just making a note that these filenames will be updated in the near future to adhere to EE2 conventions.